rector.php

```diff
@@ -50,6 +51,12 @@
 
     $rectorConfig->parallel();
 
+    // Github action cache
+    $rectorConfig->cacheClass(FileCacheStorage::class);
+    if (is_dir('/tmp')) {
```

Comment:
Should this be `/tmp/rector`? Or will it create on demand?

Reply:
that `/rector` will be created when not exists, but `/tmp` path may not be exists in some OS, eg: windows

https://github.com/rectorphp/rector-src/blob/25f3af626f841f7e62606a3a9aa64789a460f74c/src/Caching/CacheFactory.php#L35-L37

of course, the better is make is ci detected verify part of RectorConfig API.

https://github.com/rectorphp/rector-src/blob/25f3af626f841f7e62606a3a9aa64789a460f74c/config/config.php#L32

but currently prefixed and used internally.